### PR TITLE
fixed doclint config for javadoc

### PR DESCRIPTION
**Pull request by patduin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## TBD
+### Changed
+- Disabled javadoc doclint correctly for this plugin version.
+
+
```
Comment (AnanaMJ) [inline CHANGELOG.md]: There's an extra blank line

**Review by AnanaMJ — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -382,7 +382,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
             <configuration>
-              <additionalparam>-Xdoclint:none</additionalparam>
+              <doclint>none</doclint>
```
Comment (massdosage) [inline pom.xml:384]: I think you should now also be able to remove line 117 - `<compilerArgument>-Xlint:deprecation</compilerArgument>` for the `maven-compiler-plugin`. Can you please confirm?

**Review by massdosage — COMMENTED** [review]
Just one comment from me, if you can check whether that can also be removed now.

File: pom.xml (hunk 2 above)
Comment (patduin) [reply pom.xml:384]: looks like it can go

**Review by massdosage — APPROVED** [review]

**Pull request merged**